static/app/data/platformCategories.tsx

```diff
@@ -94,6 +94,7 @@ export const backend = [
   'ruby-rails',
   'ruby-rack',
   'rust',
+  'kotlin',
```

Comment:
I don't know if `kotlin` is a valid platform here since I only see the Kotlin when choosing "All" on UI but the 'all' const does not exist, please review :)

Reply:
It maps to the docs platform strucutre. It'll load this one: https://github.com/getsentry/sentry-docs/tree/master/src/wizard/kotlin

Reply:
are you sure? eg ruby-rack isn't a valid folder name under src/wizard

Reply:
Does. Ruby folder, has a rack.md:

https://github.com/getsentry/sentry-docs/blob/master/src/wizard/ruby/rack.md

Reply:
`'platform':
`wizard/platform/index.md` 
or `platform-something`:
`wizard/platform/something.md`

Reply:
ah ok, didn't know it'd work for children's folder too